Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java

```diff
@@ -128,6 +128,19 @@ public Element build(final CallbackProperties properties) throws SAMLAssertionBu
 
             // add attribute statements
             final Subject evidenceSubject = createEvidenceSubject(properties, certificate, publicKey);
+            // add conditions statement
+            DateTime beginValidTime = new DateTime();
```

Comment:
This doesn't allow for values to be passed in.  Should discuss the appropriate solution for this.